### ISSUE TITLE
adding value to missing twice should not work, independently of UI

### ISSUE THREAD

**Issue by surchs** [neurobagel/annotation_tool#171]
This is an old draft, filing it here as a test case:

> If I mark a column-value as "missing" and then do it again (e.g. click button twice), the second time I try this, something should catch that the value is already declared missing (I actually thought I had a catch for this)

**Comment by surchs** [comment]
The idea here is to write a regression test for this case, and then to close the issue when it passes.

**Comment by jarmoza** [comment]
Assigning this to myself as I just implemented `changeMissingStatus`. This can easily be prevented by ensuring only unique values are added to the `missingValues` list for a column in the `dataDictionary`.

**Comment by surchs** [comment]
Yeah, I think this issue was left so we can check that the bug doesn't come back. Thanks for addressing it!

**Comment by jarmoza** [comment]
Closed by #365
